pkgs/by-name/ms/msnake/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ncurses,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "msnake";
+  # last release tag was 13 years ago
+  version = "unstable-2025-02-09";
```

```suggestion
  version = "0.1.1-unstable-2025-02-09";
```
See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning